### ISSUE TITLE
FORMAT

### ISSUE THREAD

**Issue by Howon** [ADI-Labs/mixtapes.io#9]
please do this:
```
npm install -g jshint
jshint (whatever js file you are submitting)
```

so we have a uniform format

**Comment by Howon** [comment]
@liunick @francesar @chanpham97 @kyshim

**Comment by alanhdu** [comment]
Yes. Please.

Also, can y'all do some rebaseing and squashing of commits? It's super hard to follow the logic of the commits as-is (especially when there are a bunch of merge conflicts between branches xP).
